.github/workflows/push-pr-lint.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v4
       with:
-        go-version: '^1.19'
+        go-version: '^1.21'
```

Comment:
what's wrong with 1.22?

Reply:
I like the 1.22 for loops decision https://tip.golang.org/doc/go1.22 they say that it maintains the Go 1 [promise of compatibility](https://tip.golang.org/doc/go1compat)

Reply:
Yeah and it would be useful for at least one of the things I fixed in this PR, but I’m in no hurry tbh.